CONTRIBUTING.md

```diff
@@ -30,6 +30,10 @@ JsBarcode has tests on all barcode symbologies (and more). If you want to test y
 
 There are also visual tests located under ``test/browser/``.
 
+Using Docker
+----
+If you have installed [Docker](https://www.docker.com), you can avoid local installation of dependencies for [node-canvas](https://github.com/Automattic/node-canvas). For doing that, run ``npm install --ignore-scripts`` instead of ``npm install``. Then you can run tests by ``docker-compose up``
```

Comment:
Running any type of npm command outside of docker is not necessary?

Reply:
`npm install` will fail if you don't have installed node-canvas' dependencies.